packages/core/src/modules/dids/domain/key-type/ed25519.ts

```diff
@@ -40,6 +44,10 @@ export const keyDidEd25519: KeyDidMapping = {
       return getKeyFromJsonWebKey2020(verificationMethod)
     }
 
+    if (isMultikey(verificationMethod)) {
```

Comment:
We also have this in keyDidMapping (a higher level), so i don't know when it'll reach this, but i see we have the same duplication for JsonWebKey2020 so it's okay for now.

We can refactor it a bit later on